src/ontology/cl-edit.owl

```diff
@@ -32194,6 +32196,29 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30382198") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4042013 "Lamp5 Lhx6 neuron"@en)
 SubClassOf(obo:CL_4042013 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002202 obo:UBERON_0004026) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032148) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032533)))
 
+# Class: obo:CL_4042014 (spiny VIP neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") obo:IAO_0000115 obo:CL_4042014 "A VIP GABAergic cortical interneuron with a soma located in L1-3 of some neocortex in Mmus. This neuron has a multipolar morphology with spiny dendrites concentrating on L1 of the cortex, and has a burst firing electrophysiological signature with highly dynamic dendritic spines.")
+AnnotationAssertion(terms:contributor obo:CL_4042014 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042014 "2024-07-11T12:09:53Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") oboInOwl:hasExactSynonym obo:CL_4042014 "superficial VIP-positive spiny neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35418660") oboInOwl:hasRelatedSynonym obo:CL_4042014 "multipolar VIP neuron")
+AnnotationAssertion(rdfs:label obo:CL_4042014 "spiny VIP neuron"@en)
+SubClassOf(obo:CL_4042014 ObjectIntersectionOf(obo:CL_4023016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001950)))
```

Comment:
We normally use the intersections for equivalent to, but for this I would have two separate axioms.